app/modules/key_management/secret_manager.py

```
@@ -21,13 +23,38 @@
 class SecretManager:
     @staticmethod
     def get_client_and_project():
-        if os.getenv("isDevelopmentMode") == "disabled":
+        """Get Secret Manager client and project ID based on environment."""
+        is_dev_mode = os.getenv("isDevelopmentMode", "enabled") == "enabled"
```

Ruff (0.8.2)

Use capitalized environment variable `ISDEVELOPMENTMODE` instead of `isDevelopmentMode`

(SIM112)